apps/tsri/pages/profile/index.tsx

```diff
@@ -43,7 +43,10 @@ function Profile() {
         </SubscriptionLink>
       </ProfileHeading>
 
-      <PersonalDataFormContainer mediaEmail="info@tsri.ch" />
+      <PersonalDataFormContainer
+        mediaEmail="info@tsri.ch"
+        fields={['firstName', 'address', 'password', 'image']}
```

Comment:
Missing _name, zip, city, country, email_ compared to now

![image](https://github.com/user-attachments/assets/5b9b7f7f-671f-4ca7-b92f-935307697659)

Reply:
![image](https://github.com/user-attachments/assets/57bc2992-29ce-4561-8906-bcb7b75ddacc)
![image](https://github.com/user-attachments/assets/728527a0-5168-4591-82b9-dc293cc87bb3)

did you check the wrong forms?